app/cdap/components/GridTable/index.tsx

```diff
@@ -184,9 +186,19 @@ export default function GridTable() {
     getGridTableData();
   }, [gridData]);
 
+  const showRecipePanelHandler = () => {
+    setShowRecipePanel((prev) => !prev);
+  };
+
   return (
     <Box>
+      <button onClick={showRecipePanelHandler} data-testid="recipe-count">
+        recipe count
```

Comment:
recipe count should come from translation

Reply:
Done